modules/sql/src/main/scala/gem/sql/enum/F2Enums.scala

```diff
@@ -34,7 +34,7 @@ object F2Enums {
       },
 
       EnumDef.fromQuery("F2ReadMode", "Flamingos2 read modes") {
-        type R = Record.`'tag -> String, 'shortName -> String, 'longName -> String, 'description -> String, 'minimumExposureTime -> Duration, 'recommendedExposureTime -> Duration, 'readoutTime -> Duration, 'readCount -> Int, 'readNoise -> Double`.T
+        type R = Record.`'tag -> String, 'shortName -> String, 'longName -> String, 'description -> String, 'minimumExposureTime -> FiniteDuration.Milliseconds, 'recommendedExposureTime -> FiniteDuration.Milliseconds, 'readoutTime -> FiniteDuration.Milliseconds, 'readCount -> Int, 'readNoise -> Double`.T
```

Comment:
We should add the database migration and store these as seconds in a `numeric` while we're at it.